odkx-src/trouble-shooting.rst

```diff
@@ -0,0 +1,38 @@
+.. spelling::
+    
+  Tecno
+  Infinix
+
+
+Troubleshooting
+=====================
+
+In this guide, we cover the likely problems you *might* encounter installing and configuring the ODK-X Basic Tools. The ODK-X Basic Tools are ODK-X Services, ODK-X Tables and ODK-X Survey. 
+
+Do ensure that you have followed all the instructions for installation stated in the :doc:`basics-install` page.
+
+.. _troubleshoot-devices:
+
+Devices
+--------------------
+
+This section covers issues that are specific to the Android device you are working with and how to solve these problems.
+
+.. note::
+  The ODK-X Basic Tool require an Android device with a version 4.4 or higher operating system. 
+    
+Certain **Tecno** and **Infinix** devices run specific system settings which prevents applications from running in the background to conserve battery life and prevent the device from lagging, especially on low memory devices.

```

Comment:
prevent rather than prevents. Not sure about "lagging" here? Maybe just end sentence after conserve battery life and then explain ODK-X won't work if it can't run in the background.